flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/TestingJobManagerSharedServicesBuilder.java

```diff
@@ -54,7 +54,7 @@ public TestingJobManagerSharedServicesBuilder() {
 		scheduledExecutorService = TestingUtils.defaultExecutor();
 		libraryCacheManager = ContextClassLoaderLibraryCacheManager.INSTANCE;
 		restartStrategyFactory = new NoOrFixedIfCheckpointingEnabledRestartStrategyFactory();
-		stackTraceSampleCoordinator = mock(StackTraceSampleCoordinator.class);
+		backPressureSampleCoordinator = mock(BackPressureSampleCoordinator.class);
```

Comment:
It is better not to mock class here. Since we already touched this class change, then it should be properly used.